test/elixir_bench_web/schema/schema_test.exs

```diff
@@ -38,26 +21,45 @@ defmodule ElixirBenchWeb.SchemaTest do
         "data" => %{
            "repos" => [
              %{
-               "name" => "ecto",
-               "owner" => "tallysmartins",
-               "slug" => "tallysmartins/ecto"
+               "name" => repo.name,
+               "owner" => repo.owner,
+               "slug" => repo.owner <> "/" <> repo.name
+             },
+             %{
+               "name" => another_repo.name,
+               "owner" => another_repo.owner,
+               "slug" => another_repo.owner <> "/" <> another_repo.name
```

Comment:
nothing big, I just usually think that interpolation "#{value}/#{value}" can be neater :)

Reply:
Check! We can follow this pattern from now.